src/sentry/conf/server.py

```diff
@@ -1554,6 +1554,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:performance-new-trends": False,
     # Enable debug views for trendsv2 to be used internally
     "organizations:performance-trendsv2-dev-only": False,
+    # Bypass 30 day date range selection when fetching new trends data
+    "organizations:performance-trends-new-data-date-range-default": False,
```

Comment:
I think you might need to ship the feature flag separately before using it

Reply:
Sounds good, I've put up a separate PR here https://github.com/getsentry/sentry/pull/54719